README.md

```diff
@@ -16,29 +17,39 @@
     </p>
 </div>
 
-<!-- ABOUT THE PROJECT -->
-
 ## About The Project
-
 Miranum-Connect is a library which enables developers to build reusable and technology neutral connectors and integrations between products.
-Please find our official docs <a href="https://miranum.com/docs/components/miranum-connect/intro-miranum-connect">here</a>.
-<p align="right">(<a href="#top">back to top</a>)</p>
+More information can be found in the [official documentation](https://miranum.com/docs/components/miranum-connect/intro-miranum-connect).
+
+Content of this readme:
+* [Version Compatibility](#version-compatibility)
+* [Miranum-Worker](#miranum-worker)
+* [Miranum-Process](#miranum-process)
+* [Miranum-Message](#miranum-message)
+* [Contribution](#contributing)
+* [Code of Conduct](#code-of-conduct)
+* [License](#license)
+
+## Version Compatibility
+| Miranum Connect Version | JDK  | Camunda Platform Versions | Compatible Spring Boot versions  |
+|-------------------------|------|---------------------------|----------------------------------|
+| >=0.1.0                 | > 11 | >=8.0, >=7.15             | >= 2.7.6, 3.0.x                  |
 
 ## Get Started
-
-Miranum-Connect is currently available as [snapshot in maven-central](https://s01.oss.sonatype.org/content/repositories/snapshots/io/miragon/miranum/).
-You can import all java moduls as dependencies. For Example:
+To use the Java Miranum-Connect library, declare the following Maven dependency in your project:
 
 ```xml
 <dependency>
-    <groupId>io.miragon.miranum</groupId>
-    <artifactId>json-api</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
+ <groupId>io.miragon.miranum</groupId>
+ <artifactId>connect</artifactId>
```

Comment:
you can't download directories. A dependency needs to be a valid java package.

Reply:
A first list would be: worker-api, message-api, process-api, json-api, task-api 
I have added this to the description.  👍